hmda/src/main/scala/hmda/persistence/submission/HmdaValidationError.scala

```diff
@@ -213,7 +213,7 @@ object HmdaValidationError
         if (state.statusCode == Verified.code) {
           val timestamp = Instant.now().toEpochMilli
           val signed = SubmissionSigned(submissionId, timestamp, Signed)
-          if (state.qualityVerified && state.macroVerified) {
+          if ((state.qualityVerified && state.macroVerified) || state.statusCode == Verified.code) {
```

Comment:
We check `state.statusCode == Verified.code` in an `if` statement a few lines above, so this `if` statement will now always return true.

Reply:
Well that's embarrassing , I'll rectify this asap  @nickgrippin .